image/assets/lang/English.json

```diff
@@ -343,15 +343,17 @@
   "app_settings_display_wallpaper_clock": "Clock",
   "app_settings_display_wallpaper_quotes": "Quotes",
   "app_settings_display_wallpaper_edit_quotes": "Edit quotes",
-  "app_settings_display_wallpaper_quotes_edit": "Edit quote",
-  "app_settings_display_wallpaper_quotes_delete": "Delete quote",
+  "app_settings_display_wallpaper_edit_custom_quotes": "Edit custom quotes",
+  "app_settings_display_wallpaper_quotes_edit": "Edit",
+  "app_settings_display_wallpaper_quotes_delete": "Delete",
   "app_settings_display_wallpaper_quotes_new": "New quote",
-  "app_settings_display_wallpaper_quotes_delete_confirmation": "Delete quote?",
+  "app_settings_display_wallpaper_quotes_delete_confirmation": "Delete this quote?",
+  "app_settings_display_wallpaper_quotes_delete_success": "Quote deleted.",
   "app_settings_display_wallpaper_quotes_note": "Note",
   "app_settings_display_wallpaper_quotes_author": "Author",
   "app_settings_display_wallpaper_quotes_our_favourites": "Our favourites",
   "app_settings_display_wallpaper_quotes_custom": "Custom",
-  "app_settings_display_wallpaper_quotes_categories": "Categories",
+  "app_settings_display_wallpaper_quotes_categories": "Select categories",
```

Comment:
@mjan-dev How to handle translations? I have created new keys but beside English/Polish the translations has to be updated. I have left english/old strings in other languages.

Reply:
Translation updated by strings provided from translation office